mac-setup.sh

```diff
@@ -51,7 +51,7 @@ echo
 echo "Running Khan Installation Script 1.2"
 
 if ! sw_vers -productVersion 2>/dev/null | grep -q '10\.1[12345]\.' ; then
```

Comment:
@dbraley you wrote this originally.  Do we need to be updating this regexp?  I think we likely do.

FWIW, I found this table of product-versions:
```
14    Sonoma      2023
13    Ventura     2022
12    Monterey    2021
11    Big Sur     2020  (may appear as 10.16)
10.15 Catalina    2019
10.14 Mojave      2018  last 32-bit support
10.13 High Sierra 2017
10.12 Sierra      2016
10.11 El Capitan  2015
10.10 Yosemite    2014
10.9  Mavericks   2013
10.8  Mountain Lion 2012
10.7  Lion        2011
10.6  Snow Leopard 2009
10.5  Leopard     2007
10.4  Tiger       2005  first 64-bit support
10.3  Panther     2003
10.2  Jaguar      2002
10.1  Puma        2001
10.0  Cheetah     2001
```
So maybe the regexp was meant to capture people installing on something later than Catalina?  But it also captures yosemite and earlier as well, perhaps by mistake.

My guess is you should change this to be `grep -q '^1[0-4]\.'`.  Which matches everything that's been released so far, but not 15.x (whenever that comes out).  Since it seems like this is only intended to warn on new OSes, not obsolete old ones.

And then change line 57 to say "please update this message and the grep that produces it" or something like that.

Reply:
Done and done. Please take another look when you get the chance. Thanks!